### ISSUE TITLE
Containers variable return null object

### ISSUE THREAD

**Issue by Jimz71** [AsYetUntitled/Framework#601]
### Steps to reproduce the behaviour
1. 
2. 
3. 
4.

**Comment by BoGuu** [comment]
Seems you've omitted important parts of the issue template, could you please correct that?

**Comment by DomT602** [comment]
@Jimz71 Hi, sorry for the delay in responding, can I just clarify, are you saying that the variable "containers" which is set on the house is returning objNull, or is it an array containing objNull's? Also is this after adding then removing a container?

**Comment by DreaMz71** [comment]
@DomT602 Yes containers var containing objNull's when add or delete and after disconnect,
So the count of containers is not what it should be when you try to place another container.

**Comment by DomT602** [comment]
Hi @Jimz71, above you can see 2 PR's to resolve this issue. #652 for the 5.x.x branch, and #653 for master branch. Please could you confirm if this resolves your issue?

**Comment by cryptearth** [comment]
As already noted: #653 fixes the dup-bug for current master branch. I didn't checkecd #652 for 5.x.x branch.
